src/MaIN.Services/Utils/CurlRequestParser.cs

```diff
@@ -0,0 +1,158 @@
+﻿using MaIN.Domain.Entities.Agents.AgentSource;
```

Comment:
For comments (in whole file)
We documment code in english, and also keep in mind that those comments should be here as a help to understand it better for maintenance in future. So if you find some debugging leftovers, remove them :)